content/credits.yaml

```diff
@@ -37,6 +37,19 @@ team:
   - Kristina Plummer
   - Britannio Jarrett
   - Kieran Uddin
+  - Amanda Lin
+  - Angela Yeung
+  - Karen Wong
+  - Aishwarya Vardhana
+  - Alex Hays
+  - David Kaneda
+  - Diana Alayon
+  - Graeme Fordyce
+  - Kellie Menendez
+  - Kristina Plummer
+  - Spencer Gabor
+  - Chris Hosmer
+  - Vivian Cromwell
```

Comment:
```suggestion
  - Vivian Cromwell
  - Shilpa Apte
```

Reply:
Sorry I will have to do it in a different PR